migrations/20230131115800-index-paypal-transaction-id.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    // Copy all IDs to the new standard field (took 40s against a local prod dump)
+    await queryInterface.sequelize.query(`
+      UPDATE "Transactions"
+      SET "data" = JSONB_SET(
+          "data",
+          '{paypalCaptureId}',
+          TO_JSON(
+            COALESCE(
+              "data" -> 'capture' ->> 'id',
+              "data" -> 'paypalSale' ->> 'id',
+              "data" -> 'paypalTransaction' ->> 'id'
+            )::text
+          )::jsonb
+        )
+      WHERE "data" ->> 'paypalCaptureId' IS NULL
+      AND (
+        "data" -> 'capture' ->> 'id' IS NOT NULL
+        OR "data" -> 'paypalSale' ->> 'id' IS NOT NULL
+        OR "data" -> 'paypalTransaction' ->> 'id' IS NOT NULL
+      )
+    `);
+
+    // Create index
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "transactions__data_paypal_capture_id"
+      ON "Transactions"
+      USING BTREE (("data"->>'paypalCaptureId') ASC)
```

Comment:
```suggestion
      USING BTREE (("data"#>>'{paypalCaptureId}') ASC)
```

I've been using this form to match the same operator Sequelize uses in its query because I had an index failing in the past.

Reply:
Interesting, I've used it in many places and never faced any issues so far. Would be great to document the exact problem.

I'll commit the change to be safe

Reply:
I'll make a proper test for this. Sorry about causing fear.